main.go

```diff
@@ -97,12 +97,19 @@ func (m *MozProcessStat) Diff(data MozProcessStat) {
 	m.NetworkIO.Fifoout -= data.NetworkIO.Fifoout
 }
 
+// SystemMemoryInfo summarises information about the system memory usage
+type SystemMemoryInfo struct {
+	TotalMemory      uint64  `json:"vmem_total`
+	TotalSwap        uint64  `json:"swap_total"`
+	AvailableMemory  uint64  `json:"vmem_available`
+	UsedPercent      float64 `json:"vmem_used_percent`
```

Comment:
@srfraser would it make sense to ceil/floor this value?

Reply:
Is an int preferable? I'm not sure

Reply:
Good point, I've switched this to `int` instead.